### ISSUE TITLE
Warnings emitted for deprecated features in django (`DjangoRemovedIn40`)

### ISSUE THREAD

**Issue by dustinlharrison** [jazzband/dj-rest-auth#116]
# Symptom

Using pytest to run unit tests in my project triggers a `DjangoRemovedIn40` warning after the release of Django 3.1 when using the `dj_rest_auth.registration.url` package.

``` logs
/usr/local/lib/python3.7/site-packages/dj_rest_auth/registration/urls.py:22: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
```

Should be able to trigger using a TestCase Client to make a URL call to a registered path.

**Comment by mohmyo** [comment]
Thanks for raising this issue, I was just thinking about submitting a PR to use path() and re_path() instead of deprecated url(), but I find out that, from django 1.11 is supported and path() was only introduced starting from Django 2.0.

So, what do you think? @iMerica

**Comment by iMerica** [comment]
Yeah sounds good to use `path()`. This package is targeting Django 2+.

**Comment by iMerica** [comment]
I think there are a few Django 3.1 errors now so if you can submit a PR on this one, I'll look at the others and we can cut a 3.1 compatibility release to Pypi that has all the fixes.

**Comment by mohmyo** [comment]
Great, I will work on this now.

**Comment by iMerica** [comment]
Fixed in #119
